sw/device/lib/testing/lc_ctrl_testutils.h

```diff
@@ -18,4 +18,13 @@
  */
 bool lc_ctrl_testutils_debug_func_enabled(const dif_lc_ctrl_t *lc_ctrl);
 
+/**
+ * Check if Lifecycle Controller count number is expected.
+ *
+ * This function will read out lc_transition_cnt register and check the value
+ * against exp_lc_count value.
+ */
+void check_lc_state_transition_count(const dif_lc_ctrl_t *lc_ctrl,
```

Comment:
To stay consistent with the [style guide](https://docs.opentitan.org/sw/device/lib/testing/) while also keeping this name short, I might suggest:
 
```suggestion
void lc_ctrl_testutils_check_transition_count(const dif_lc_ctrl_t *lc_ctrl,
```

Reply:
Thanks TIm! Will update the that.